### PR TITLE
Edits to .Net Connector PR5592

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx [hunk 1]
```diff
@@ -249,10 +249,10 @@ Depending on your application type, you might need to import the namespace into
 
 ##### .NET Entity Framework Core
 
-To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is either:
+To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is:
 
 `C:\Program Files\edb\dotnet\EF.Core\EFCore.PG\net8.0`
-  
+<!-- This said "either" but only has one option>
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx:255]:
```suggestion

```

File: product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx [hunk 2]
```diff
@@ -249,10 +249,9 @@ Depending on your application type, you might need to import the namespace into
 
 ##### .NET Entity Framework Core
 
-To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is either:
+To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is:
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx:252]: This said "either" but only has one option

File: product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx [hunk 3]
```diff
@@ -7,48 +7,48 @@ title: "Installing and configuring the .NET Connector"
...
+For more information, see the [EDB .NET Connector Now Published on NuGet](https://www.enterprisedb.com/blog/improving-developer-experience-updated-edb-net-connector-now-published-nuget) blog post.
 
 #### Install NuGet package via Visual Studio interface
 
-- Right-click your project or solution and choose "Manage NuGet packages"
-- Search the package using "enterprisedb.edbclient" as search text
-- Choose the "EnterpriseDB.EDBClient" package
-- Click "Install" to proceed to package download and installation
+1. Right-click your project or solution and select **Manage NuGet package**.
+1. Search the package using `enterprisedb.edbclient as the search text.
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx:35]:
```suggestion
1. Search the package using `enterprisedb.edbclient` as the search text.
```

File: product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx [hunk 4]
```diff
@@ -101,13 +101,13 @@ For information about configuring the .NET Connector in each environment, see:
 
 To reference library files with Microsoft Visual Studio:
 
-1.  Select the project in the Solution Explorer.
+1.  In the Solution Explorer, select the project.
 2.  Select **Project > Add Reference**.
 3.  In the Add Reference` dialog box, browse to select the appropriate library files.
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx:106]:
```suggestion
3.  In the Add Reference dialog box, browse to select the appropriate library files.
```

File: product_docs/docs/net_connector/8.0.2.1/07_retrieving_database_records.mdx [hunk 5]
```diff
@@ -62,7 +62,7 @@ private void Page_Load(object sender, System.EventArgs e)
 
 ```
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/07_retrieving_database_records.mdx:64]: Is there an encoding error? The text that follows is appearing in blue.

File: product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx [hunk 6]
```diff
@@ -5,7 +5,7 @@ title: "Using a ref cursor in a .NET application"
 
 <div id="using_a_ref_cursor_in_a_net_application" class="registered_link"></div>
 
-A `ref cursor` is a cursor variable that contains a pointer to a query result set. The result set is determined by executing the `OPEN FOR` statement using the cursor variable. A cursor variable isn't tied to a particular query like a static cursor. You can open the same cursor variable a number of times with the `OPEN FOR` statement containing different queries and each time. A new result set is created for that query and made available by way of the cursor variable. You can declare a cursor variable in two ways:
+A `ref cursor` is a cursor variable that contains a pointer to a query result set. The result set is determined by executing the `OPEN FOR` statement using the cursor variable. A cursor variable isn't tied to a particular query like a static cursor. You can open the same cursor variable a number of times with the `OPEN FOR` statement containing different queries each time. A new result set is created for that query and made available by way of the cursor variable. You can declare a cursor variable in two ways:
```
Comment (ebgitelman) [inline product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx:8]: Please check this edit, as I wasn't sure.

File: product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx:252]: yes it has only option now

File: product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx (hunk 6 above)
Comment (nidhibhammar) [reply product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx:8]: @moazzumali, can you please confirm?

Comment (moazzumali) [reply product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx:8]: @nidhibhammar Yes. correct.

File: product_docs/docs/net_connector/8.0.2.1/07_retrieving_database_records.mdx (hunk 5 above)
Comment (nidhibhammar) [reply product_docs/docs/net_connector/8.0.2.1/07_retrieving_database_records.mdx:64]: yeah, the code block is cpp

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**